server/konnectors/weboob.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+/**
+ * A Konnector to communicate with a Weboob instance.
+ *
+ * You need a running [cozyweboob](https://github.com/Phyks/cozyweboob)
+ * somewhere.
+ *
+ * // TODO: What happens when moment is an invalid date?
```

Comment:
Yes, typically if whatever module did not manage to retrieve a date for the bill. A bill will almost always have a date, and this should not be an issue, but it may not have duedate / startdate / finishdate. Should these just be set to "null" ? "undefined" ?